django-spectre-css/templates/spectre-css/render-multi-field-part.html

```diff
@@ -0,0 +1,21 @@
+{% load widget_tweaks_extras %}
+{% if bare %}
+    {% render_multi_field field index class+=additional_class class+=field_class id=field_id %}
+{% else %}
+    <div class="form-group">
+        <label class="form-label" for="{{ field_id }}">{{ label_text }}{# field.label #}</label>
+        {% if field.form.is_bound %}
+            {% if field.errors %}
+                {% render_multi_field field index class+=additional_class class+="is-error" class+=field_class id=field_id %}
+                {% include "spectre-css/render_errors.html" %}
+            {% else %}
+                {% render_multi_field field index class+=additional_class class+=field_class class+="is-success" id=field_id %}
+            {% endif %}
+        {% else %}
+            {% render_multi_field field index class+=additional_class class+=field_class id=field_id %}
+        {% endif %}
+        {% if field.index.help_text %}
+            <small class="form-text text-muted">{{ field.index.help_text|safe }}</small>
```

Comment:
We should make the warning in the README a bit more visible and not just place it within the example section.

Reply:
I have moved the notice to its own section and added that one should not put user generated input into it.